archivers/upx-devel/Portfile

```diff
@@ -0,0 +1,44 @@
+# $Id$
```

Comment:
Please remove the `# $Id$` line; we aren't using this anymore now that we've moved from Subversion to git. But do please add the [standard modeline](https://guide.macports.org/chunked/development.creating-portfile.html).